services/121-service/src/payments/payments.service.ts

```diff
@@ -52,6 +53,7 @@ import { MappedPaginatedRegistrationDto } from '@121-service/src/registration/dt
 import { ReferenceIdsDto } from '@121-service/src/registration/dto/reference-id.dto';
 import { DefaultRegistrationDataAttributeNames } from '@121-service/src/registration/enum/registration-attribute.enum';
 import { RegistrationStatusEnum } from '@121-service/src/registration/enum/registration-status.enum';
+import { RegistrationDataScopedRepository } from '@121-service/src/registration/modules/registration-data/repositories/registration-data.scoped.repository';
```

Comment:
RegistrationData conceptually does not exist anymore. Refactor as part of Kobo Toolbox integration?

Reply:
I added them in scope of the Epic.